snakemake/uf_cells_spleen_0510/src/convert_h5ad_to_arrow.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+import argparse
+
+from anndata import read_h5ad
+import pyarrow as pa
+import pandas as pd
+
+
+def h5ad_to_arrow(h5ad_file, arrow_file):
+    ann_data = read_h5ad(h5ad_file)
+    umap = ann_data.obsm['X_umap'].transpose()
+    leiden = ann_data.obs['leiden'].to_numpy().astype('uint8')
```

Comment:
Would the categorical type work here? Maybe it would be harder to use on the client.

Reply:
Sorry, could cast once it is a dataframe.

Reply:
Not sure, i copied this code from portal-containers https://github.com/hubmapconsortium/portal-containers/blob/master/containers/h5ad-to-arrow/context/main.py#L15